packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -198,21 +198,21 @@ function NetworkContainer({
 
     if (isArbitrumNova) {
       return {
-        backgroundImage: `url('/images/ArbitrumNovaLogo.svg')`,
-        backgroundClassName: 'bg-[#8a4100]'
+        backgroundImage: `url('/images/TransparentNovaLogo.webp')`,
```

Comment:
Any reason we're not using svg anymore? The images look simple enough that we could use svg